assets/_sass/custom-viz/boeing/_base.scss

```diff
@@ -18,6 +18,30 @@
 
     svg {
       height: 100%;
+      width: 100%;
+    }
+
+    .boeing-counter {
+      display: flex;
+      justify-content: center;
+
+      li {
+        font-family: $font_inconsolata;
+        list-style: none;
+        text-transform: uppercase;
+        font-size: 1rem;
```

Comment:
This should be set using the `fontSize` mixin so it's consistent with the rest of the site.